doc/en/user/source/community/geopkg/output.rst

```diff
@@ -22,32 +22,36 @@ GeoPackage WPS Process
 
 A custom GeoPackage can be created with any number of tiles and features layers using the ``GeoPackage`` WPS Process (see :ref:`wps_processes`).
 
-The WPS process takes in one parameter: ``contents`` which is an XML schema that represents the desired output.
-
-General outline of a ``contents`` scheme::
-
-      <geopackage name=”mygeopackage” xmlns="http://www.opengis.net/gpkg">
 
-      <features name=”myfeaturelayer” identifier=”L01”>
-	      <description>describe the layer</description>
-	      <srs> EPSG:4216 </srs>
-	      <bbox>
-		      <minx>-180</minx>
-		      <miny>-90</miny>
-		      <maxx>180</maxx>
-		      <maxy>90</maxy>
-	      </bbox>
-	  ...
-      </features>
+.. warning:: While the the process generates a compliant GeoPackage, some abilities like generalization, style and part of the metadata export
+   are based on unofficial extensions discussed in the `Testbed 16 GeoPackage engineering report <http://docs.opengeospatial.org/per/20-019r1.html>`_.
 
-      <tiles name=”mytileslayer” identifier=”L02”>
-	      <description>describe the layer</description>
-	      <srs>..</srs>
-	      <bbox>..</bbox>
-	      ...
-      </tiles>
+The WPS process takes in one parameter: ``contents`` which is an XML schema that represents the desired output.
 
-      </geopackage>
+General outline of a ``contents`` scheme:
+
+.. code-block:: xml
+
+    <geopackage name="mygeopackage" xmlns="http://www.opengis.net/gpkg">
+        <features name="myfeaturelayer" identifier="L01">
+            <description>describe the layer</description>
+            <srs>EPSG:4216</srs>
```

Comment:
Not critical for this example but... is 4216 the right code?
Wondering if there was a keyboard offset under the fingers so it was a 32 instead of a 21 (so 4326 instead of 4216)...

... just kidding :)

Reply:
No clue, I've just added proper XML highlight and formatting for this section, was already there. I'd a legit EPSG code anyways :-D